iceoryx_posh/include/iceoryx_posh/mepoo/chunk_header.hpp

```diff
@@ -101,10 +99,18 @@ struct alignas(32) ChunkHeader
     /// @return the chunk size
     uint32_t chunkSize() const noexcept;
 
+    /// @brief The size of the chunk occupied by the user-header
+    /// @return the user-header size
+    uint32_t userHeaderSize() const noexcept;
+
     /// @brief The size of the chunk occupied by the user-payload
     /// @return the user-payload size
     uint32_t userPayloadSize() const noexcept;
 
+    /// @brief The alignment of the chunk occupied by the user-payload
+    /// @return the user-payload alignment
+    uint32_t userPayloadAlignment() const noexcept;
+
     /// @brief The unique identifier of the publisher the chunk was sent from
     /// @return the id of the publisher the chunk was sent from
     UniquePortId originId() const;
```

Comment:
no deal breaker - If you touch this PR again could you please add noexcept to all remaining methods.